isisd/isis_srv6.h

```diff
@@ -17,7 +17,6 @@
 #define ISIS_DEFAULT_SRV6_MAX_END_POP_MSD         3
 #define ISIS_DEFAULT_SRV6_MAX_H_ENCAPS_MSD        2
 #define ISIS_DEFAULT_SRV6_MAX_END_D_MSD           5
-#define ISIS_DEFAULT_SRV6_IFNAME                 "sr0"
```

Comment:
if you remove the define, you will break the compilation.
it is highly recommended to avoid breaking compilation.
I recommend you to squash this commit with the commit next to this commit that adds the define in lib/srv6.h file.